CalcCurrentValue/CalcCurrentValue/CalcCurrentValue.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CalcCurrentValue
+{
+    class CalcCurrentValue
+    {
+        public enum TransactionType
```

Comment:
It is better to have new types in a separate file. Please move enum from Program.cs to TransactionType.cs. You can add a new file by right clicking on the project name and then selecting "add" > "class..."